modules/javafx.graphics/src/test/java/test/javafx/scene/SceneTest.java

```diff
@@ -971,4 +973,52 @@ public void testAddNullPostLayoutPulseListener() {
         SceneShim.scenePulseListener_pulse(scene);
         assertEquals(1, counter.get());
     }
+
+    @Test public void testNoReferencesRemainToRemovedNodeAfterBeingClicked() {
+        StubToolkit toolkit = (StubToolkit) Toolkit.getToolkit();
```

Comment:
This cast isn't necessary, is it?

Reply:
Yes, it is necessary, since it won't compile otherwise.